ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template5.kt

```diff
@@ -170,10 +173,16 @@ private fun ColumnScope.Template5MainContent(
 @Composable
 private fun HeaderImage(uri: Uri?) {
     uri?.let {
+        val aspectRatio = windowAspectRatio()
```

Comment:
What if the paywall isn't displayed full screen and is instead a dialog?

Reply:
Hmm I was assuming this would pick the window size of the dialog, not the screen, but I was wrong, 
so this change would still apply there. As long as the width of the screen is bigger than the height, we will set the image as a percentage of the screen, instead of using an aspect ratio. This is not ideal, so trying to figure out a solution. Great catch! Thanks!

Reply:
I changed the algorithm to use the template size instead of the window size, so we don't care if it's displayed as a dialog, as full screen, or anything the developer wants to do. This will cause a recomposition unfortunately... But I think it's the best solution